### ISSUE TITLE
Several Opera versions flagged as “current”

### ISSUE THREAD

**Issue by meduzen** [mdn/browser-compat-data#6174]
I suppose each browser can only have one version with `current` as status, as the [schema states](https://github.com/mdn/browser-compat-data/blob/master/schemas/browsers-schema.md#release-objects):
> current: This release is the official latest release.

However, in [`browsers/opera.json`](https://github.com/mdn/browser-compat-data/blob/847bdf29462e99568696ac6abf09acc13bb3fed2/browsers/opera.json#L471), Opera 57, 63 and 67 are flagged as `current`:

```json
{
  "browsers": {
    "opera": {
      "name": "Opera",
      "releases": {

        "57": {
          "release_date": "2018-11-28",
          "release_notes": "https://dev.opera.com/blog/opera-57/",
          "status": "current",
          "engine": "Blink",
          "engine_version": "70"
        },

        "63": {
          "release_date": "2019-08-20",
          "release_notes": "https://blogs.opera.com/desktop/2019/08/opera-63-initial-release/",
          "status": "current",
          "engine": "Blink",
          "engine_version": "76"
        },

        "68": {
          "release_date": "2020-04-22",
          "release_notes": "https://blogs.opera.com/desktop/2020/04/opera-68-is-here-with-built-in-instagram-in-the-sidebar/",
          "status": "current",
          "engine": "Blink",
          "engine_version": "81"
        },

      }
    }
  }
}
```

Other browsers are okay.

**Comment by vinyldarkscratch** [comment]
This was just fixed in #6168.  I’ve submitted a pull request to make sure this no longer happens in the future!

**Comment by meduzen** [comment]
Ha, fantastic! I’ve extensively searched in the issues, but not in the PR. 😅
